data/Dockerfiles/acme/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine:3.6
+
+LABEL maintainer "Andre Peters <andre.peters@servercow.de>"
+
+RUN apk add --update --no-cache \
+      acme-client \
+      docker \
```

Comment:
Here is the API docs - https://docs.docker.com/engine/api/v1.29/#operation/ContainerRestart